demos/free_surface/implicit_cylindrical_free_surface.py

```diff
@@ -0,0 +1,56 @@
+from gadopt import *
+from gadopt.utility import CombinedSurfaceMeasure
+from implicit_free_surface import ImplicitFreeSurfaceModel
+from test_viscous_surface import run_benchmark
+
+
+class CylindricalImplicitFreeSurfaceModel(ImplicitFreeSurfaceModel):
+    # Free surface relaxation test in a cylindrical domain.
+
+    name = "implicit-cylindrical-iterative"
+    bottom_free_surface = False
+
+    def __init__(self, dt_factor, do_write=True):
+        self.cartesian = False
+        super().__init__(dt_factor, do_write=do_write, cartesian=self.cartesian)
```

Comment:
Maybe the following pattern is a bit more general (also keeps `do_write` at its false default)

```suggestion
    def __init__(self, dt_factor, **kwargs):
        super().__init__(dt_factor, cartesian=False, **kwargs)
```

Reply:
ok!